cpp/test/sg/ridge.cu

```diff
@@ -72,31 +72,55 @@ class RidgeTest : public ::testing::TestWithParam<RidgeInputs<T>> {
     rmm::device_uvector<T> labels(params.n_row, stream);
     T alpha = params.alpha;
 
+    /* How to get the coefficients:
```

Comment:
Same here- just a little more descriptive so we know that this is how to get valid coefficients for testing (if that's the purpose).